src/map/besieged_data.h

```diff
@@ -0,0 +1,97 @@
+#ifndef SERVER_BESIEGED_DATA_H
+#define SERVER_BESIEGED_DATA_H
+
+#include "common/cbasetypes.h"
+#include "common/sql.h"
+
+enum BESIEGED_STRONGHOLD
+{
+    ALZAHBI,
```

Comment:
Is this internal, should these have values associated with them? (ALZAHBI would default to 0, but its better to make that obvious)

Reply:
ie. are these entirely internal, or are they linked to packets or the client etc.?

Reply:
They are linked to packets

Reply:
Added indexes for robustness.